com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -281,6 +281,7 @@ void Update()
                             {
                                 VFXGraph.compileReporter = reporter;
                                 AssetDatabase.ImportAsset(AssetDatabase.GetAssetPath(graphView.controller.model));
+                                graph.SetExpressionGraphDirty(false); // As are implemented subgraph now, compiling dependents chain can reset dirty flag on used subgraphs, which will make an infinite loop, this is bad!
```

Comment:
I realized we assumed the previous `AssetDatabase.ImportAsset` was synchronous here. I don't think it's possible to have an asynchronous import of the vfx anyway.